frontend/src/pages/Projects/Project/ProjectDetails/Deliverables/index.tsx

```diff
@@ -6,7 +6,7 @@ import { useRenderTableCell } from "components/PLAYGROUND/hooks/useRenderTableCe
 import { useFormControls } from "hooks";
 import { useFormData } from "hooks/useFormData";
 import { IFormControls } from "types";
-import { tableConfig } from "./tableConfig";
+import { tableConfig } from "./TableConfig";
```

Comment:
I keep seeing this. My git settings are updated as recommended by @ASpiteri-BCGov. I guess the file could be lowercased instead of changing the import. 

either way, every time I pull development, this bug crops up.